### ISSUE TITLE
train loss is huge, map nearly = 0

### ISSUE THREAD

**Issue by congjianting** [balancap/SSD-Tensorflow#135]
i train my own data, find the loss is very huge , and test the train data ,map is nearly = 0.  could anyone give me some tips what has happened? 

('', <open file '/tmp/tfmodel/training_config.txt', mode 'w' at 0x1116b1660>)
INFO:tensorflow:Evaluating ./logs/model.ckpt-75361
INFO:tensorflow:Starting evaluation at 2017-09-17-14:19:54
INFO:tensorflow:Restoring parameters from ./logs/model.ckpt-75361
INFO:tensorflow:Evaluation [1/50]
INFO:tensorflow:Evaluation [2/50]
INFO:tensorflow:Evaluation [3/50]
INFO:tensorflow:Evaluation [4/50]
INFO:tensorflow:Evaluation [5/50]
INFO:tensorflow:Evaluation [6/50]
INFO:tensorflow:Evaluation [7/50]
INFO:tensorflow:Evaluation [8/50]
INFO:tensorflow:Evaluation [9/50]
INFO:tensorflow:Evaluation [10/50]
INFO:tensorflow:Evaluation [11/50]
INFO:tensorflow:Evaluation [12/50]
INFO:tensorflow:Evaluation [13/50]
INFO:tensorflow:Evaluation [14/50]
INFO:tensorflow:Evaluation [15/50]
INFO:tensorflow:Evaluation [16/50]
INFO:tensorflow:Evaluation [17/50]
INFO:tensorflow:Evaluation [18/50]
INFO:tensorflow:Evaluation [19/50]
INFO:tensorflow:Evaluation [20/50]
INFO:tensorflow:Evaluation [21/50]
INFO:tensorflow:Evaluation [22/50]
INFO:tensorflow:Evaluation [23/50]
INFO:tensorflow:Evaluation [24/50]
INFO:tensorflow:Evaluation [25/50]
INFO:tensorflow:Evaluation [26/50]
INFO:tensorflow:Evaluation [27/50]
INFO:tensorflow:Evaluation [28/50]
INFO:tensorflow:Evaluation [29/50]
INFO:tensorflow:Evaluation [30/50]
INFO:tensorflow:Evaluation [31/50]
INFO:tensorflow:Evaluation [32/50]
INFO:tensorflow:Evaluation [33/50]
INFO:tensorflow:Evaluation [34/50]
INFO:tensorflow:Evaluation [35/50]
INFO:tensorflow:Evaluation [36/50]
INFO:tensorflow:Evaluation [37/50]
INFO:tensorflow:Evaluation [38/50]
INFO:tensorflow:Evaluation [39/50]
INFO:tensorflow:Evaluation [40/50]
INFO:tensorflow:Evaluation [41/50]
INFO:tensorflow:Evaluation [42/50]
INFO:tensorflow:Evaluation [43/50]
INFO:tensorflow:Evaluation [44/50]
INFO:tensorflow:Evaluation [45/50]
INFO:tensorflow:Evaluation [46/50]
INFO:tensorflow:Evaluation [47/50]
INFO:tensorflow:Evaluation [48/50]
INFO:tensorflow:Evaluation [49/50]
INFO:tensorflow:Evaluation [50/50]
2017-09-17 22:21:17.862489: I tensorflow/core/kernels/logging_ops.cc:79] AP_VOC07/mAP[0.00050391871943065015]
2017-09-17 22:21:18.345659: I tensorflow/core/kernels/logging_ops.cc:79] AP_VOC12/mAP[0.00045786099240169385]
Time spent : 84.253 seconds.
INFO:tensorflow:Finished evaluation at 2017-09-17-14:21:18
Time spent per BATCH: 1.685 seconds.

**Comment by KKKlose** [comment]
do you know how to output the map value?,thank you
